examples/02-javascript-basics/helpers/array_equality.js

```diff
@@ -0,0 +1,19 @@
+Array.isEqual = function (array1, array2) {
```

Comment:
@paulomilitante Please move this file to `examples/helpers/` as this can be shared across modules.

Reply:
@adimasuhid 
I have moved the directory of /helpers and removed $.filter() content